python/orca/src/bigdl/orca/learn/pytorch/callbacks/maincallback.py

```diff
@@ -91,3 +93,29 @@ def on_val_forward(self, runner):
         Any behavior inconsistent with the default training behavior should be overridden here.
         """
         self.on_iter_forward(runner)
+
+    def before_train_iter(self, runner):
+        features, target = runner.batch
+        if torch.is_tensor(features):
+            runner.batch = features, target
+        elif isinstance(features, (tuple, list)):
+            runner.batch = *features, target
```

Comment:
I don't think this is a good idea to expose this to users, for these lines are intended to restore `batch` to the format users expects, see pr https://github.com/intel-analytics/BigDL/pull/5763
You may discuss whether runner.batch could be unpack in  `features, target = runner.batch`, if not then we assume it is a custom data format like a single dict {'x': x,'y':y}.